openapi/external.yaml

```diff
@@ -0,0 +1,161 @@
+openapi: 3.0.1
+info:
+  version: 1.0.0
+  title: IO Lollipop Function Lollipop Consumer API
+  x-logo:
+    url: https://io.italia.it/assets/img/io-logo-blue.svg
+  description: >
+    Documentation of the IO Lollipop Function API exposed to Lollipop
+    Consumerhere.
+servers:
+  - url: https://api.pagopa.it/api/v1/lollipop
+security:
```

Comment:
@fabriziopapi commented this https://github.com/pagopa/io-infra/pull/436/files#r1108467618, this is the right point for that comment.